API_Auto/src/test/java/testGetBrandList/BrandsTestCases.java

```diff
@@ -0,0 +1,55 @@
+package testGetBrandList;
+
+import base.CrudOperation;
+import io.restassured.path.json.JsonPath;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import payload.Brand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BrandsTestCases extends CrudOperation {
+
+    private static final Logger log = LogManager.getLogger("Brands.class");
```

Comment:
Please check your classname and give it accordingly

Reply:
Done